main/main.cpp

```diff
@@ -991,6 +991,10 @@ Error Main::setup(const char *execpath, int argc, char *argv[], bool p_second_ph
 			debug_collisions = true;
 		} else if (I->get() == "--debug-navigation") {
 			debug_navigation = true;
+#endif
+#if defined(DEBUG_ENABLED)
```

Comment:
While you're here, we killed `SERVER_ENABLED` and `main.cpp` has the last few remaining reference to it. You could remove them and thus keep this in the same `#if defined(DEBUG_ENABLED)` as the other debug flags.

Reply:
The command-line help should be updated to mention `--debug-stringnames` under the Debug options section.

Reply:
Its kind of an internal debug flag, not sure if it makes a lot of sense to expose to users